specs/linux_file_system_hierarchy.md

```diff
@@ -80,6 +80,11 @@ and should fall back to defaults if configuration is missing,
 following the
 [Configuration Files Specification](configuration_files_specification.md).
 
+If `/opt/` is supported (see below),
+then the `/etc/opt/` subdirectory is the location where third-party software installed in `opt` store their
```

Comment:
Is `/etc/opt/` a thing for anything but chrome? I was about to ask whether it was a thing at all, but then found a machine with a directory `/etc/opt/chrome`. I'm asking because I wonder whether it makes sense to spec, since software using `/opt/` will most likely not care for any other standards either.